content/v2.0/query-data/guides/monitor-states.md

```diff
@@ -148,9 +148,9 @@ Detect state changes with the `monitor.stateChanges()` function. To use the `mon
 
     {{< nav-icon "alerts" >}}
 
-2. If you haven't already, [create a check](/v2.0/monitor-alert/checks/create/) that stores statuses (`CRIT`, `WARN`, `INFO`, `OK` or `ANY`) in the `_level` column. <!-- specify how to do this with monitor.check() function or in UI, with check threshold or deadman? 
+2. If you haven't already, [create a check](/v2.0/monitor-alert/checks/create/) that stores statuses (`CRIT`, `WARN`, `INFO`, `OK` or `ANY`) in the `_level` column. <!-- specify how to do this with monitor.check() function or in UI, with check threshold or deadman?
```

Comment:
comment still a question?

Reply:
@kelseiv I just updated and/or commented out broken links in this doc. This was commented in your original draft of this doc.